Base/Logic/vtkSlicerApplicationLogic.h

```diff
@@ -32,6 +32,7 @@
 
 // STL includes
 #include <mutex>
+#include <thread>
```

Comment:
I am following up with the following updates:

```diff
// ITK includes
- #include <itkPlatformMultiThreader.h>
+ #include <itkThreadSupport.h> // For itk::ITK_THREAD_RETURN_TYPE
```

and updating the comment associated the docstrings of these callbacks:

```diff
-   /// Callback used by a MultiThreader to start a processing thread
+  /// Callback used by a std::thread to start a processing thread
  static itk::ITK_THREAD_RETURN_TYPE ProcessingThreaderCallback( void * );

-   /// Callback used by a MultiThreader to start a networking thread
+  /// Callback used by a std::thread to start a networking thread
  static itk::ITK_THREAD_RETURN_TYPE NetworkingThreaderCallback( void * );
```